tests_utils/test-cn-stalone-Docker.sh

```diff
@@ -0,0 +1,51 @@
+#! /bin/bash -x
```

Comment:
this should be rewritten to take into account the Makefile that has been added in the meantime